app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,50 @@
+class InventoriesController < ApplicationController
+  def index
+    @user = current_user
+    @inventories = Inventory.all
+  end
+
```

Comment:
The method gives an error because there is no current user available. You could add a condition to prevent the error. Example
`@user = current_user if current_user.present?`